### DISCUSSION TITLE
Understanding daisyUI semantic theme colors

### DISCUSSION THREAD

**Discussion by ChristopherHButler** [saadeghi/daisyui#3024] [Q&A] [answered]
I have read through the daisyUI documentation on [colors ](https://daisyui.com/docs/colors/)and it's entirely possible I missed it somewhere, but I am trying to understand how the neutral color is decided. It seems like it is base-200 using the [theme generator](https://daisyui.com/theme-generator/), could that be correct?

**Comment by saadeghi** [comment, marked as answer]
When you need a dark, unsaturated color which stays dark on both dark and light themes, you can use `neutral`  
An example would be the code block on this page: https://daisyui.com/docs/install/ We want that to be a dark block on both light and dark themes. We can't just use a negative color (like `base-100`/`base-content`) because a big white block on a dark theme is not ideal.

Let me know if you have a question.

**Reply by ChristopherHButler** [reply]
Thanks for the quick response. 
I think I meant to say **default** color, not _neutral_. It seems like the default color is base-200 using the [theme generator](https://daisyui.com/theme-generator/)? That's not quite intuitive to me.
